lang/en_us.php

```diff
@@ -210,6 +210,7 @@
 	'InvalidSecurityCode'     => 'Please enter the security code correctly.',
 	'InvalidPassword'         => 'Your password contains invalid characters.',
 	'InvalidBirthdate'        => 'Invalid birthdate input.',
+	'AntiRegesterationSpam'   => 'Anti Registeration Spam System is Active , Contact the Admin to Complete your registeration.',
```

Comment:
'You already have a pending registration. Please check your mails and follow the confirmation process.'